src/App.jsx

```diff
@@ -1,34 +1,24 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import Header from './components/Header'
+import { Routes, Route } from 'react-router-dom';
+import Articles from './components/Articles';
+import Homepage from './components/Homepage';
+import SingleArticle from './components/SingleArticle';
 
-function App() {
-  const [count, setCount] = useState(0)
 
+function App() {
+  
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+   <>
+   <Header />
+   <Routes>
+    <Route path='/home' element={<Homepage/>}/>
```

Comment:
we should also have a route for "/" - this can be the homepage too. We dont really need the /home path